src/tests/ftest/util/dmg_utils.py

```diff
@@ -263,7 +263,7 @@ def storage_set_faulty(self, uuid, force=True):
         return self._get_json_result(
             ("storage", "set", "nvme-faulty"), uuid=uuid, force=force)
 
-    def storage_query_list_devices(self, rank=None, health=False):
+    def storage_query_list_devices(self, rank=None, health=False, uuid=None):
```

Comment:
minor - we should add an args entry for the new param
```
uuid (str, optional): device UUID. Defaults to None.
```

Reply:
Huh, I thought I did that must've got lost in committing. Will add as doc change after CI run or if I have to repush.

Reply:
updated